src/govuk/components/task-list/_index.scss

```diff
@@ -0,0 +1,103 @@
+@include govuk-exports("govuk/component/task-list") {
+  .govuk-task-list {
+    @include govuk-font($size: 19);
+    margin-top: 0;
+    margin-bottom: 0;
+    padding: 0;
+    list-style-type: none;
+    @include govuk-media-query($from: tablet) {
+      min-width: 550px;
+    }
+  }
+
+  .govuk-task-list__items {
+    @include govuk-font($size: 19);
+    @include govuk-responsive-margin(9, "bottom");
+    padding-left: 0;
+    list-style: none;
+    @include govuk-media-query($from: tablet) {
+      padding-left: govuk-spacing(6);
+    }
+  }
+
+  .govuk-task-list__item {
+    margin-bottom: 0;
+    padding-top: govuk-spacing(2);
+    padding-bottom: govuk-spacing(2);
+    border-bottom: 1px solid $govuk-border-colour;
+    @include govuk-clearfix;
+  }
+
+  .govuk-task-list__item:first-child {
+    border-top: 1px solid $govuk-border-colour;
+  }
+
+  .govuk-task-list__task-name {
+    display: block;
+    @include govuk-media-query($from: 450px) {
+      float: left;
+    }
+  }
+
+  .govuk-task-list__hint {
+    display: block;
+    margin-top: 5px;
+    color: $govuk-secondary-text-colour;
+  }
+
+  .govuk-task-list__status {
+    display: inline-block;
+    margin-top: govuk-spacing(2);
+    margin-bottom: govuk-spacing(1);
+    margin-left: 0;
+    padding-top: 2px;
+    padding-right: 4px;
+    padding-bottom: 2px;
+    padding-left: 0;
```

Comment:
What is the reason for setting this to `0`? I have attached a screenshot to show the padding being lost on smaller screens creating an imbalance.

![image](https://user-images.githubusercontent.com/11505395/163403273-cc0244c3-cf48-434b-9ac3-a1c97a4b590f.png)

Reply:
Well spotted - this is a bug, will fix!